recipes/cis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: cis
+  version: "1.3.2"
+
+source:
+  #path: ../../cis
+  git_url: https://github.com/cedadev/cis.git
+  git_tag: 1.3.2
```

Comment:
Neat, I didn't know you could do that. Why not use the git_tag hook instead though? It saves having to update the hash every time.